src/v2/controllers/FormController.js

```diff
@@ -71,32 +75,34 @@ export default Controller.extend({
     if (_.isFunction(actionFn)) {
       // TODO: OKTA-243167
       // 1. what's the approach to show spinner indicating API in fligh?
-      // 2. how to catch error?
       actionFn()
         .then(this.handleIdxSuccess.bind(this))
         .catch(error => {
-          throw error;
+          this.showFormErrors(this.formView.model, error);
         });
     } else {
-      throw `Invalid action selected: ${actionPath}`;
+      const errorResp = {
+        errorSummary: `Invalid action selected: ${actionPath}`,
```

Comment:
does it make sense to show actionPath in the error to enduser ? I don't think it's necessary.

Reply:
I think this is more for handling edge case where it shall never happens to enduser. If it does, surface detail might help troubleshooting?
Alternatively we can do log.

Reply:
I think keeping just the sentence should be sufficient for enduser.. `Invalid action selected. Please <do something>`... and we can log the action path detail in console